tests/Support/SupportArrTest.php

```diff
@@ -5,7 +5,7 @@
 use stdClass;
 use ArrayObject;
 use Tightenco\Collect\Support\Arr;
-use Tightenco\Collect\Support\Carbon;
+use Carbon\Carbon;
```

Comment:
This is correct using the Tightenco-namespaced Carbon, as in the original tests.

Reply:
Yes, that's right. I will revert this.